src/main/kotlin/car/io/ResultView.kt

```diff
@@ -0,0 +1,19 @@
+package car.io
+
+import car.business.Cars
+
+class ResultView(val cars: Cars) {
+    fun print() {
+        for (i in cars.getPositions()) {
+            printCarPosition(i)
+        }
+        println()
+    }
+
+    private fun printCarPosition(count: Int) {
+        for (i in 0..count) {
+            print("-")
+        }
+        println()
```

Comment:
`String`에 대해 호출할 수 있는 `repeat()`가 있습니다.

```suggestion
        println("-".repeat(count))
```

Reply:
String에서 지원하는 함수를 한번 읽어봐야겠군요!